schemas/context/opportunity-crm.schema.json

```diff
@@ -0,0 +1,136 @@
+  ],
+  "description": "CRM specific opportunity fields.",
+  "definitions": {
+    "opportunity-crm": {
+      "properties": {
+        "xdm:name": {
+          "title": "Opportunity Name",
+          "description": "Subject or descriptive name, such as the expected order or company name, for the opportunity.",
+          "type": "string"
+        },
+        "xdm:description": {
+          "title": "Opportunity Description",
+          "description": "Additional information to describe the opportunity, such as possible products to sell or past purchases from the customer.",
+          "type": "string"
+        },
+        "xdm:stage": {
+          "title": "Opportunity Stage",
+          "description": "Sales stage of this opportunity to aid the sales team in their efforts to win this opportunity.",
+          "type": "string"
+        },
+        "xdm:amount": {
+          "title": "Opportunity Amount",
+          "description": "Opportunity Amount.",
+          "type": "number"
+        },
+        "xdm:quantity": {
+          "title": "Opportunity Quantity",
+          "description": "Opportunity Quantity.",
+          "type": "number"
+        },
+        "xdm:fiscalYear": {
+          "title": "Fiscal Year",
+          "description": "Fiscal Year this opportunity is targeted.",
+          "type": "string"
+        },
+        "xdm:fiscalQuarter": {
+          "title": "Fiscal Quarter",
+          "description": "Fiscal Quarter this opportunity is targeted..",
+          "type": "string"
+        },
+        "xdm:hasOpportunityLineItem": {
+          "title": "Has Opportunity Line Item",
+          "description": "Flag to indicate whether this opportunity have more line items.",
+          "type": "boolean"
+        },
+        "xdm:probabilityPercentage": {
+          "title": "Probability Percentage",
+          "description": "Likelihood that opportunity will close, stated as a percentage.",
+          "type": "number",
+          "minimum": 0,
+          "maximum": 100
+        },
+        "xdm:isClosed": {
+          "title": "Closed",
+          "description": "Flag indicating if this opportunity is closed.",
+          "type": "boolean"
+        },
+        "xdm:isWon": {
+          "title": "Won",
+          "description": "Flag indicating if this opportunity is Won.",
+          "type": "boolean"
+        },
+        "xdm:closeDate": {
+          "title": "Close Date",
+          "description": "Date of closure for this opportunity.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:nextStep": {
+          "title": "Next Step",
+          "description": "Description of next task in closing opportunity.",
+          "type": "string"
+        },
+        "xdm:lastActivity": {
+          "title": "Last Activity",
+          "description": "Last activity captured for this sales opportunity.",
+          "type": "string"
+        },
+        "xdm:isprivate": {
```

Comment:
we should remove the `is` prefix here. If we decide not to `P` should be capitalized.